script_facade/rxnav/transcode.py

```diff
@@ -0,0 +1,50 @@
+import requests
+
+
+rxnav_system_map = {
+    'NDC': 'http://hl7.org/fhir/sid/ndc',
+    'rxnorm': 'http://www.nlm.nih.gov/research/umls/rxnorm',
+}
+
+def to_rxnorm(coding, rxnav_url):
+    """Return first RxNorm coding to given NDC coding"""
+    reverse_system_map = {v: k for k, v in rxnav_system_map.items()}
+    querystring_params = {
+        'idtype': reverse_system_map[coding['system']],
```

Comment:
should you protect from KeyError, in the event of an unknown `coding['system']`?

Reply:
Both PDMPs we've integrated only return NDC; fair critique, but I don't think we'll see anything but NDC-coded drugs